jetty-core/jetty-deploy/src/test/java/org/eclipse/jetty/deploy/MockAppProvider.java

```diff
@@ -60,9 +60,9 @@ public ContextHandler createContextHandler(App app) throws Exception
 
         String name = app.getPath().toString();
         name = name.substring(name.lastIndexOf("-")  + 1);
-        File war = new File(webappsDir, name);
+        Path war = new File(webappsDir, name).toPath();
```

Comment:
Why don't you change the `File webappsDir` field instead?

Reply:
Done.